pkg/cluster/inventory.go

```diff
@@ -402,7 +402,8 @@ func (i *DefaultInventory) ClustersToReconcile(reconcileInterval time.Duration)
 
 func (i *DefaultInventory) ClustersNotReady() ([]*State, error) {
 	statusFilter := &statusFilter{
-		allowedStatuses: []model.Status{model.ClusterStatusReconciling, model.ClusterStatusReconcileFailed, model.ClusterStatusError},
+		// TODO: ask about allowedStatuses
+		allowedStatuses: []model.Status{model.ClusterStatusReconciling, model.ClusterStatusReconcileFailed, model.ClusterStatusError, model.ClusterStatusReconcileDisabled},
```

Comment:
Where is this method used? If its used to pick up clusters for next cycle than probably we shouldn't be returning those for which reconciliation was disabled

Reply:
This method is used only to filter clusters to expose the right Prometheus metrics. 
[here](https://github.com/kyma-incubator/reconciler/blob/874713c60a30b8a95870fdb4c9b460e6d667ca17/pkg/metrics/reconciliation_not_ready.go#L40)